enterprise/backend/test/metabase_enterprise/audit_app/permissions_test.clj

```diff
@@ -31,6 +34,9 @@
   (mt/test-drivers #{:postgres :h2 :mysql}
     (audit-test/with-audit-db-restoration
       (mt/with-premium-features #{:audit-app}
+        (is (= "complete"
```

Comment:
i'm surprised this poll-until needs to be here. Why does this block when in testing?

Reply:
This might be left in accidentally, I'll remove it